plugin/evm/vm.go

```diff
@@ -826,6 +829,10 @@ func (vm *VM) Shutdown(context.Context) error {
 		log.Error("error stopping state syncer", "err", err)
 	}
 	close(vm.shutdownChan)
+	// Stop RPC handlers before eth.Stop which will close the database
+	for _, handler := range vm.rpcHandlers {
+		handler.Stop()
```

Comment:
In the Stop comment it says ` waits for stopPendingRequestTimeout to allow pending requests to finish`; but I guess that's just a stale comment. It seems they have removed it around v1.4.0 (8 years ago lol).

Reply:
I think it still has some impact since it calls close on the codecs [here](https://github.com/ava-labs/subnet-evm/blob/e996852950081016b8a1117b9a4d3d6200fc0072/rpc/server.go#L194)
but I couldn't find that variable either